pre-docs/inventory_version2.md

```diff
@@ -541,6 +541,80 @@ plugins:
 * `private-key`: The path to the private key file. default: `keys/private_key.pkcs7.pem`
 * `public-key`: The path to the public key file. default: `keys/public_key.pkcs7.pem`
 
+#### Vault plugin
+
+
+This plugin allows config values to be set by accessing secrets on a Vault server. It supports several fields:
```

Comment:
Would it be helpful to have a link out to some external resource explaining what Vault is? I've never heard of it, but this sounds like a great feature that I'd want to take advantage of. Or maybe we should just rely on users to engage their own google-fu to find out more?

Reply:
I'm not sure. I'll ask about adding a link out during standup tomorrow. Thanks for the suggestions, Chris!